README.md

```diff
@@ -1,7 +1,6 @@
 # ipcam-view ![ipcam-view](images/logo.png)
 
 [![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-ipcam--view-brightgreen.svg?style=flat)](http://android-arsenal.com/details/1/3358)
-[![Download](https://api.bintray.com/packages/niqdev/maven/mjpeg-view/images/download.svg)](https://bintray.com/niqdev/maven/mjpeg-view/_latestVersion)
```

Comment:
Are you going to add JitPack in a different PR?

Reply:
It's less work, when you do it. You have the power to edit `README.md` directly

Please add this info https://jitpack.io/#niqdev/ipcam-view/v1.6.0

<img width="723" alt="image" src="https://user-images.githubusercontent.com/3314607/112767880-65b6e800-9019-11eb-86b8-09fe7b0280b8.png">

This is the batch
`
[![](https://jitpack.io/v/niqdev/ipcam-view.svg)](https://jitpack.io/#niqdev/ipcam-view)`

Reply:
yup, I'll take care of that